modules/samples/artifacts/0033/publisher.sh

```diff
@@ -2,6 +2,7 @@
 for i in {1..50}; do
```

Comment:
We can remove this file. And ass instructions to run it using the simulator in the editor.